IoTuring/Logger/consts.py

```diff
@@ -56,7 +56,7 @@
 # before those spaces I add this string
 LONG_MESSAGE_PRESTRING_CHAR = ' '
 
-CONSOLE_LOG_LEVEL = "LOG_INFO"
+CONSOLE_LOG_LEVEL = "LOG_DEBUG"
```

Comment:
Do not change this. You can override the log level for development with an envvar:
```shell
env IOTURING_LOG_LEVEL=LOG_DEBUG IoTuring
```

https://github.com/richibrics/IoTuring#debug-log